model/communication/Endpoint.ttl

```diff
@@ -51,6 +51,14 @@ ids:AppEndpoint
     # rdfs:subClassOf ids:Endpoint ; #TODO: not possible. See explanation above.
     rdfs:label "DataApp Endpoint"@en ;
     rdfs:comment "Description of endpoints a DataApp offers."@en ;
+    idsm:validation [
+        idsm:forProperty ids:appEndpointPort;
+        idsm:constraint idsm:NotNull ;
```

Comment:
The port declaration could be optional, as some might use default ports. You choose

Reply:
Good point! Havent thought about that.